src/Entity.php

```diff
@@ -585,4 +585,23 @@ protected function getResolvedFields(): array
     {
         return static::fields();
     }
+
+    /**
+     * Make new entity instance with copied fields
+     *
+     * @return Entity
+     */
+    public final function copy(): Entity
```

Comment:
Prettier is used in this project, `final` comes first.

Just a thought, would it make sense for an entity to override this method?